### PR TITLE
fix the triangle exercise

### PR DESCRIPTION
**Pull request by Radix16**
#399

### CODE REVIEW

**Comment by rchavarria** [comment]
Merging #459 conflicts with this (and probably #458). A workaround could be to randomly select exercises to fix, separated exercises in `.eslintignore`, so that `git` is able to merge them without conflicts.

Meanwhile, would you mind to fix conflicts?

File: exercises/triangle/example.js [hunk 1]
```diff
@@ -22,8 +22,10 @@ function Triangle(a, b, c) {
   };
 
   this.violatesInequality = function () {
-    var a = this.sides[0], b = this.sides[1], c = this.sides[2];
-    return (a + b < c) || (a + c < b) || (b + c < a);
+    var sideA = this.sides[0];
```
Comment (rchavarria) [inline exercises/triangle/example.js]: When triangle is created, `a`, `b` and `c` params are assigned to `this.sides`. Is there really a need to *extract* them into separate variables? In fact, would `return (a + b < c) || (a + c < b) || (b + c < a);` work without redeclaring `a`, `b` and `c`?

Just asking.

Comment (Radix16) [reply exercises/triangle/example.js]: @rchavarria You are absolutely correct.  I will alter the code.

File: .eslintignore [hunk 2]
```diff
@@ -36,5 +36,4 @@ exercises/secret-handshake
 exercises/simple-cipher
 exercises/simple-linked-list
 exercises/triangle
```
Comment (rchavarria) [inline .eslintignore]: Just another thing. Maybe I'm mixing up PRs. Shouldn't this PR remove `triangle` from `.eslintignore`?

**Comment by Radix16** [comment]
@rchavarria Do you have a tip to correct the merge conflicts?  Would it help if I simply deleted one of the two PRs?  I tried the approach described at https://help.github.com/articles/resolving-a-merge-conflict-on-github/, but the conflict remains.

**Comment by Radix16** [comment]
@rchavarria I am still not sure how to correct the merge conflicts between this PR and #458.    I tried the online GitHub merge editor as suggested in Should I just delete one of these two PRs and wait for the other to be merged?

**Comment by rchavarria** [comment]
Maybe, resolving a conflict and changing the same line (more or less) creates a new conflict.

There are very few changes, and we're developers, we fix things by resetting them 🤣 . What about creating a new PR?

**Comment by rchavarria** [comment]
Just testing the GitHub tool to resolve conflicts online. It seems to work fine. I'll move forward this PR.

**Comment by rchavarria** [comment]
Thanks @Radix16 for the contribution and sorry to make you waiting for so long.

**Pull request merged**